### PR TITLE
List index feature

### PR DESCRIPTION
**Pull request by Nakirendral**
Trying out a few changes and updates made to the project to get http://localhost:3000/lists working!

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -2,36 +2,26 @@
...
-    t.integer "user_id"
-    t.datetime "created_at", precision: 6, null: false
-    t.datetime "updated_at", precision: 6, null: false
-  end
-
-  create_table "users", force: :cascade do |t|
-    t.string "name"
-    t.string "email"
-    t.datetime "created_at", precision: 6, null: false
-    t.datetime "updated_at", precision: 6, null: false
```
Comment (FifthSurprise) [inline db/schema.rb:34]: This looks like you lost a migration somewhere and dropped users.  Not sure if it's intentional but also it's fine to add back later.

File: app/views/lists/index.html.erb [hunk 2]
```diff
@@ -1 +1,24 @@
-<h1 class="title">Listr Index</h1>
\ No newline at end of file
+<h1 class="title">Listr Index</h1>
```
Comment (FifthSurprise) [inline app/views/lists/index.html.erb:1]: Listr?

File: app/views/lists/index.html.erb [hunk 3]
```diff
@@ -1 +1,24 @@
...
+    </tr>
+  </thead>
+
+  <tbody>
+    <% @lists.each do |list| %>
+      <tr>
+        <td><%= list.list_name %></td>
+        <td><%= link_to "Show", list %></td>
+        <td><%= link_to "Edit", edit_list_path(list) %></td>
+        <td><%= link_to "Delete", list, method: :dekete, data: { confirm: "Are you sure?" } %></td>
```
Comment (FifthSurprise) [inline app/views/lists/index.html.erb:18]: :dekete <> :delete

File: app/views/lists/index.html.erb [hunk 4]
```diff
@@ -1 +1,24 @@
...
+      </tr>
+    <% end %>
+  </tbody>
+</table>
+
+<br>
```
Comment (FifthSurprise) [inline app/views/lists/index.html.erb:24]: newline here please.

File: app/views/lists/index.html.erb (hunk 3 above)
Comment (Nakirendral) [reply app/views/lists/index.html.erb:18]: Resolved afterwards in latest branch.

File: app/views/lists/index.html.erb (hunk 2 above)
Comment (Nakirendral) [reply app/views/lists/index.html.erb:1]: Was thinking about naming it that, but changed to List. Resolved afterwards in latest branch.

File: db/schema.rb (hunk 1 above)
Comment (Nakirendral) [reply db/schema.rb]: Had to learn to add role within postGres via _psql> create role rails login;_ Added a basic Users table to the database. Resolved afterwards in latest branch.

**Pull request merged**